src/xpk/commands/cluster.py

```diff
@@ -407,6 +408,8 @@ def cluster_create(args) -> None:
       # pylint: disable=line-too-long
       f' https://console.cloud.google.com/kubernetes/clusters/details/{get_cluster_location(args.project, args.cluster, args.zone)}/{args.cluster}/details?project={args.project}'
   )
+  if args.managed_mldiagnostics:
+    _install_mldiagnostics_prerequisites()
```

Comment:
Please propagate the return code, otherwise XPK can exit cleanly while the mldiagnostics installation may fail.